stablehlo/dialect/Base.td

```diff
@@ -45,36 +45,46 @@ def HLO_Complex : Complex<AnyTypeOf<[F32, F64]>>;
 
 // TODO(b/230381284): Upstream width-specific uniform quantized element types.
 class StableHLO_UniformQuantizedSignedInt<int width>
```

Comment:
`StableHLO_UniformPerTensorQuantizedSignedInt` ?

Reply:
Revisited this:  we are using `*Quantized*` to indicate per-tensor quantization and `*QuantizedPerAxis*` to indicate per-axis quantization. this matches with MLIR conventions `quant::UniformedQuantizedType` (per-tensor) and `quant::UniformedPerAxisQuantized` (per-axis).  Will keep `StableHLO_UniformQuantizedSignedInt ` unchanged.